src/test/java/no/nav/veilarbregistrering/db/DatabaseTestContext.java

```diff
@@ -1,38 +1,11 @@
 package no.nav.veilarbregistrering.db;
 
-import lombok.val;
-import no.nav.fasit.DbCredentials;
-import no.nav.fasit.FasitUtils;
-import no.nav.fasit.TestEnvironment;
-
-import java.util.Optional;
-
-import static no.nav.fo.veilarbregistrering.config.ApplicationConfig.APPLICATION_NAME;
 import static no.nav.fo.veilarbregistrering.config.DatabaseConfig.*;
 
 public class DatabaseTestContext {
 
-    public static void setupContext(String miljo) {
```

Comment:
Ettersom vi nå alltid bruker in-memory lokalt, så var det mye konfig som var overflødig.